libs/ledger/include/ledger/protocols/main_chain_rpc_protocol.hpp

```diff
@@ -63,9 +65,10 @@ class MainChainProtocol : public service::Protocol
     return Copy(blocks);
   }
 
-  Blocks TimeTravel(Digest start, int64_t limit)
+  Travelogue TimeTravel(Digest start)
```

Comment:
I think this should have a function comment to the effect of: Given a digest, provide this block and a number of blocks preceeding this on the heaviest chain. Peers will return nothing if the digest is not on the heaviest chain, and choose how many blocks to provide.

Reply:
Except for it's not an accurate description how it currently works, they can be off the heaviest chain provided they are proof.